packages/Algernon.package/ALGCore.class/class/resetKeyBindings.st

```diff
@@ -0,0 +1,4 @@
+constants
+resetKeyBindings
```

Comment:
Is this method even used in the current code? Besides that, does it make sense to call it when we reset/cleanup Algernon?

Reply:
Actually, they don't really ever change. We only added it because one key now had to map to another function. Latest commit removes it